docker/scripts/provision-core.sh

```diff
@@ -67,8 +78,7 @@ chronos_sync() {
     echo "Sync chronos snapshot"
     aws s3 --no-sign-request sync --delete s3://injective-snapshots/$NETWORK/chronos $VOLUMES_PATH/chronos
```

Comment:
We don't need this anymore, v1 exchange had both exchange and chronos logic in same binary, which is decoupled now in indexer-exchange and indexer-chronos

Reply:
@nmarcetic they are decoupled but we still need to sync chronos snapshot right?